lib/backburner.js

```diff
@@ -26,6 +26,11 @@ export default function Backburner(queueNames, options) {
   this._debouncees = [];
   this._throttlers = [];
   this._timers = [];
+  this._eventNames = ['end', 'begin'];
+  this._eventCallbacks = {};
+  for (var i = 0; i < this._eventNames.length; i++) {
+    this._eventCallbacks[this._eventNames[i]] = [];
```

Comment:
any reason why this loop suppose more then just 2 events, we should just hardcode the ones we support as to not confuse future readers